pkg/binutils/processes.go

```diff
@@ -141,7 +150,7 @@ func StartServerProcess(log logging.Logger) error {
 	}
 	err = os.WriteFile(constants.ServerRunFile, rfBytes, perms.ReadWrite)
```

Comment:
probably we want to move ServerRunFile to .avalanche-cli?

Reply:
good catch, fixed